src/styles/Home.css

```diff
@@ -0,0 +1,30 @@
+.home {
```

Comment:
Wondering how much of this styling is unique to the home page or should be applied across all pages?

Reply:
@sarahmjenkins Not sure either, we'd have to compare our stylings for the individual pages!